core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -198,6 +198,15 @@ public interface NativeConfig {
      */
     Optional<Boolean> containerBuild();
 
+    /**
+     * Explicit configuration option to generate Position Independent Executable (PIE) native executables for Linux.
+     * If the system supports, the default behaviour is to generate native binaries without PIE for
+     * <a href="https://www.redhat.com/en/blog/position-independent-executable-pie-performance">performance reasons</a>.
+     * Some systems only support PIE executables,
+     * so this option offers the opportunity to generate native executables with PIE instead.
```

Comment:
Avoid redundant "executable" and reduce words:
```suggestion
     * Some systems, however, only support platform independent executables (PIEs),
     * this option enables the generation of such  native executables.
```